recipe/meta.yaml

```diff
@@ -23,10 +23,17 @@ requirements:
     - python
     - setuptools
     - toolchain
-
+    - backports.lzma
```

Comment:
is backports needed during build or only runtime?

Can you add the unit test that shows how it failed?